integrated_channels/moodle/transmitters/learner_data.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""
+Class for transmitting learner data to Moodle.
+"""
+
+from integrated_channels.moodle.client import MoodleAPIClient
+from integrated_channels.integrated_channel.transmitters.learner_data import LearnerTransmitter
+
+
+class MoodleLearnerTransmitter(LearnerTransmitter):
+    """
+    This endpoint is intended to receive learner data routed from the integrated_channel app that is ready to be
+    sent to Moodle.
+    """
+
+    def __init__(self, enterprise_configuration, client=MoodleAPIClient):
+        """
+        By default, use the ``MoodleAPIClient`` for learner data transmission to Moodle.
+        """
+        super(MoodleLearnerTransmitter, self).__init__(
+            enterprise_configuration=enterprise_configuration,
+            client=client
+        )
+
+    def transmit(self, payload, **kwargs):
+        """
+        Send a completion status call to Canvas using the client.
```

Comment:
Canvas? :)

Reply:
😠  why is there always one place that I forgot